### DISCUSSION TITLE
Bad readings, suggestions on positioning or settings

### DISCUSSION THREAD

**Discussion by clara-j** [jomjol/AI-on-the-edge-device#3014] [General] [answered]
I setup AI on the Edge a few weeks ago and overall it has been working great.  Some bad readings but usually corrects itself soon after with another reading.  But since one of my numbers rolled over to a 1 it keeps misreading it as a 0 and hasn't worked for a while now.  It just seems to keep thinking that digit is a 0 not a 1

Any tips on what I can do to improve this?  I don't think its the glare since that is closer to the other number.  But I was thinking of getting a side glow fibre optic cable to help with that in the future.

<img width="744" alt="Screenshot 2024-03-29 at 8 47 04 AM" src="https://github.com/jomjol/AI-on-the-edge-device/assets/13851193/7d26fa02-9602-4d12-adbf-fb55cd03970c">

**Comment by SybexX** [comment, marked as answer]
Have you tried reducing the size of the ROIs, something like the picture
![318051723-7d26fa02-9602-4d12-adbf-fb55cd03970c](https://github.com/jomjol/AI-on-the-edge-device/assets/587201/73725e43-2e27-47a4-b378-3425deaba5be)

**Comment by clara-j** [comment]
Thanks, I adjusted it as suggested and let it run all day.  Seems to be back to the way it was with some bad readings but correcting it self with later readings.
